hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableConfig.java

```diff
@@ -814,7 +814,7 @@ public String getRecordMergeStrategyId() {
   public static Triple<RecordMergeMode, String, String> inferCorrectMergingBehavior(RecordMergeMode recordMergeMode,
                                                                                     String payloadClassName,
                                                                                     String recordMergeStrategyId,
-                                                                                    String orderingFieldName,
+                                                                                    String orderingFieldNames,
```

Comment:
Should be a list?

Reply:
We only check if the preCombine fields are set in this function. So do not require a list here. Renamed to orderingFieldNamesAsString to make it more clear.